udf/zeroed_364_days_active_1_bytes.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TEMP FUNCTION
+  udf_zeroed_364_days_active_1_bytes() AS (
```

Comment:
I am not at all married to any of these function names, but this one in particular is especially awkward...

Reply:
```suggestion
  udf_one_as_365bits() AS (
```